nginx/default.conf

```diff
@@ -6,6 +6,7 @@ server {
     ssl_certificate /etc/nginx/cert.pem;
     ssl_certificate_key /etc/nginx/private.key; 
 
+    client_max_body_size 8M;
```

Comment:
Depending on who merges first, one of us will have to mirror this to the other conf in #53 (reminder to look at this at merge time)

Reply:
this line does not work anymore

Reply:
(talking about the limit line btw). I am working on a quick fix in front end and backend for this

Reply:
resolved by merging Alex's PR